MyBlog.Services/Areas/Users/UserService.cs

```diff
@@ -37,8 +37,21 @@ public async Task<UserDto> GetByIdAsync(int id)
         return userDto;
     }
 
+    public async Task<User> GetByEmailAsync(string email)
+    {
+        var user = await _context.Users.FirstOrDefaultAsync(u => u.Email == email)
+                   ?? throw new NotFoundException($"User with Email: {email} is not fount");
+        
+        return user;
```

Comment:
rename all lamda variables to understandable, for example : u => user

is not fount ))))